pkgs/by-name/dd/ddev/package.nix

```
@@ -27,8 +27,8 @@ buildGoModule rec {
 
   ldflags = [
     "-extldflags -static"
-    "-X github.com/ddev/ddev/pkg/versionconstants.DdevVersion=${version}"
-    "-X github.com/ddev/ddev/pkg/versionconstants.SegmentKey=${version}"
+    "-X github.com/ddev/ddev/pkg/versionconstants.DdevVersion=${src.rev}"
+    "-X github.com/ddev/ddev/pkg/versionconstants.SegmentKey=${src.rev}"
```

This code is a `rev -> tag` drive-by PR away from breaking

```suggestion
    "-X github.com/ddev/ddev/pkg/versionconstants.DdevVersion=v${version}"
    "-X github.com/ddev/ddev/pkg/versionconstants.SegmentKey=v${version}"
```